config/share_notify.yml

```diff
@@ -0,0 +1,3 @@
+---
+host: <%= ENV.fetch('APPS_DHH_STAGING_SHARE_NOTIFY_HOST') {'https://staging.osf.io'}%>
```

Comment:
@mcritchlow  Yes, that's why I created the sample file share_notify.yml.prod.sample and share_notify.yml.staging.sample to make it use the correct env variables. 
Since the implementation of loading the env for share gem is different than active fedora gem, I can't just make it work an easy way like solr.yml or fedora.yml.  Here's the slack messages that we discussed before about this issue  https://ucsdlibrary.slack.com/archives/its-developers/p1461792003000136